__tests__/app.test.js

```diff
@@ -41,4 +41,48 @@ describe('GET /api', () => {
             expect(endpointKeysLength).toBe(Object.keys(endpoints).length)
         })
     });
+});
+describe('All bad paths', () => {
+    test('404: should return a custom error for a bad path', () => {
+        return request(app)
+        .get('/api/notAPath')
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Not found')
+        })
+    });
+});
+describe('GET /api/articles/:article_id', () => {
+    test('200: should return an article object', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body
+            expect(article.article_id).toBe(1)
+            expect(article).toHaveProperty('author'), expect.any(String)
```

Comment:
We could make this test more robust by using the toMatchObject jest matcher as we should know exactly what values to expect for the given article https://jestjs.io/docs/expect#tomatchobjectobject